dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/converter/RelationshipTrackerConverterService.java

```diff
@@ -120,16 +120,17 @@ private org.hisp.dhis.relationship.Relationship from(
     org.hisp.dhis.relationship.RelationshipItem toItem =
         new org.hisp.dhis.relationship.RelationshipItem();
 
+    Date now = new Date();
     if (toRelationship == null) {
-      Date now = new Date();
 
       toRelationship = new org.hisp.dhis.relationship.Relationship();
       toRelationship.setUid(fromRelationship.getRelationship());
       toRelationship.setCreated(now);
-      toRelationship.setLastUpdated(now);
     }
+    toRelationship.setLastUpdated(now);
 
     toRelationship.setRelationshipType(relationshipType);
+    toRelationship.setCreatedAtClient(DateUtils.fromInstant(fromRelationship.getCreatedAtClient()));
```

Comment:
Maybe it's just Github, but the format here seems a bit off.
While we are at it, why do we not use the `Date now` variable to set the creation date here?